depmapomics/ccle.py

```diff
@@ -0,0 +1,1423 @@
+    for key, val in tocompare.items():
+        _, omissmatchCols, _, omissmatchInds, newNAs, new0s = h.compareDfs(
+            files[key], tc.get(name=TAIGA_ETERNAL, file=val)
+        )
+        print(key)
+        print(len(omissmatchCols), "columns NOT IN df1")
+        print(len(omissmatchInds), "indices NOT IN df1")
+        print(newNAs, "new NAs")
+        print(new0s, "New 0s")
+
+    print("updating the tracker")
+
+    updated_tracker = expressions.updateTracker(
+        set(renaming.keys()) - set(["transcript_id(s)"]),
+        failed,
+        lowqual[lowqual.sum(1) > 3].index.tolist(),
+        ccle_refsamples,
+        samplesetname,
+        gumbo,
+        refworkspace,
+        samplesinset=samplesinset,
+        starlogs=starlogs,
+        trackerobj=trackerobj,
+        todrop=todrop,
+        dry_run=True,
+        newgs=None,
+    )
+
+    # subset and rename, include all PRs that have associated CDS-ids
+    pr_table = track.update_pr_from_seq(trackerobj)
+    renaming_dict = dict(list(zip(pr_table.CDSID, pr_table.index)))
+    h.dictToFile(renaming_dict, folder + "rna_seq2pr_renaming.json")
+    pr_files = dict()
+    for k, v in files.items():
+        pr_files[k + "_profile"] = v[v.index.isin(set(renaming_dict.keys()))].rename(
+            index=renaming_dict
+        )
+    expressions.saveFiles(pr_files, folder)
+
+    if not dry_run:
+        print("uploading to taiga")
+        tc.update_dataset(
+            changes_description="new " + samplesetname + " release!",
+            dataset_permaname=taiga_dataset,
+            upload_files=[
+                {
+                    "path": folder + "proteincoding_genes_tpm_logp1.csv",
+                    "name": "proteincoding_genes-tpm_logp1",
+                    "format": "NumericMatrixCSV",
+                    "encoding": "utf-8",
+                },
+                {
+                    "path": folder + "transcripts_tpm_logp1.csv",
+                    "name": "transcripts-tpm_logp1",
+                    "format": "NumericMatrixCSV",
+                    "encoding": "utf-8",
+                },
+                {
+                    "path": folder + "genes_tpm_logp1.csv",
+                    "name": "genes-tpm_logp1",
+                    "format": "NumericMatrixCSV",
+                    "encoding": "utf-8",
+                },
+                {
+                    "path": folder + "genes_tpm.csv",
+                    "name": "genes-tpm_logp1",
+                    "format": "NumericMatrixCSV",
+                    "encoding": "utf-8",
+                },
+                {
+                    "path": folder + "transcripts_tpm.csv",
+                    "name": "transcripts-tpm",
+                    "format": "NumericMatrixCSV",
+                    "encoding": "utf-8",
+                },
+                {
+                    "path": folder + "proteincoding_genes_tpm.csv",
```

Comment:
for CDS id level, let's just keep the "transcripts_count" and "gene_count" files.